tests/test_folder_structure/folders_test.py

```diff
@@ -0,0 +1,15 @@
+import configparser
```

Comment:
Issue: When writing tests, please make sure to use `pytest`. In particular, this file does not contain any code that pytest will recognize as a test that needs to be ran. For starters, the test must be within a function (or class) with an appropriate name. https://docs.pytest.org/en/3.0.1/overview.html